packages/devtools-reps/src/object-inspector/utils.js

```diff
@@ -334,21 +492,30 @@ function getChildren({ getObjectProperties, actors, item }) {
   return children;
 }
 
+function getParent(item: ObjectInspectorItem) : ObjectInspectorItem | null {
+  return item.parent;
+}
+
 module.exports = {
```

Comment:
it would be nice to drop the `module.exports` in favor of `export function` style.